packages/sanity/src/core/form/field/actions/FieldActionsProvider.tsx

```diff
@@ -17,7 +28,9 @@ type FieldActionsProviderProps = PropsWithChildren<{
 export const FieldActionsProvider = memo(function FieldActionsProvider(
   props: FieldActionsProviderProps,
 ) {
-  const {actions, children, path, focused} = props
+  // by passing the comments and slot here, we can wrap this functionality around any custom field without needing to confirm the title and description
```

Comment:
huh, that's odd, do we know why?

Reply:
It seems like the FieldActionsProvider wasn't accommodating for more custom inputs like the boolean input, I wanted to be able to give them this functionality but still be able to create my own title (which is needed for the timezone input bit). This meant that the boolean input currently doesn't have the ability to be commented on or have a set of actions.